.rubocop.yml

```diff
@@ -76,6 +76,9 @@ RSpec/ContextWording:
 RSpec/StubbedMock:
   Enabled: false
 
+RSpec/IndexedLet:
```

Comment:
Any reason for this?

Reply:
Oh, yep! Sorry should I've added a comment to explain it. After bumping rubocop, they've introduced new cops, one of them triggers a few offences in our specs because it doesn't like lets named with numbers, like `date1`, `codefendant2`, etc. which we use in a few places.
I thought about renaming the lets but then I also thought it was clear enough as it was, numbered! 🤣 So for now I've disabled the cop.